lib/ex_twilio/config/env.ex

```diff
@@ -0,0 +1,91 @@
+defmodule ExTwilio.Config.Env do
+  @moduledoc """
+  This module is to use the configuration based on Application Env, kept as it is for backward compatibility if you're not using Config struct.
+  Stores configuration variables used to communicate with Twilio's API.
+
+  All settings also accept `{:system, "ENV_VAR_NAME"}` to read their
+  values from environment variables at runtime.
+  """
```

Comment:
It seems like this is more of an internal API and maybe shouldn't show up in the moduledoc. What do you think about putting this documentation in `ExTwilio.Config` and setting `@moduledoc false` in this module?

Reply:
I'll do that